apps/predictions/lib/predictions_pub_sub.ex

```diff
@@ -1,20 +1,24 @@
 defmodule Predictions.PredictionsPubSub do
   @moduledoc """
-  Allow channels to subscribe to prediction streams.
+  Allow channels to subscribe to prediction streams, which are collected into a
+  Map with keys representing route-stop pairs as "<route-id>@<parent-stop-id>"
```

Comment:
**nitpick:** this should be updated to reflect the actual implementation of `prediction_key/3`.